terraform-plans/templates/github/CONTRIBUTING.md.tftpl

```diff
@@ -0,0 +1,28 @@
+# Contributor Guide
+
+## Contributing Code
+
+### Workflow
+
+1. **Choose/Create an Issue**: Before starting work on an enhancement, create an issue that explains your use case. This helps track progress and keeps the discussion organized. The issue will be tracked on the GitHub issue page.
+
+2. **Fork the Repository**: Create a fork of the repository to make your changes.
+
+3. **Create a New Branch**: Make sure to create a new branch for your contribution.
+
+4. **Submit a Pull Request**: Submit a pull request to merge your changes into the main branch. Reference the issue by adding `#Fixed xxx` (replace `xxx` with the issue number) to automatically link the issue to your PR.
```

Comment:
I think the syntax is not correct: https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue. It may be better to directly use the link rather than giving an example

Reply:
`#xxx` or the link will work fine

Reply:
Updated to mention both will works.